charts/enterprise-kyverno-operator/templates/cr-custompolicysets.yaml

```diff
@@ -0,0 +1,21 @@
+{{- range .Values.policies.customPolicySetCharts }}
```

Comment:
Also, add commented values in the values.yaml to demonstrate how to add this, and update README to contain this new parameter.